src/js/AppStore.js

```diff
@@ -149,7 +149,7 @@ class AppStore extends React.Component {
                     image: app.iconUrl,
                     appID: app.policyAppID,
                     cmdID: app.package_url,
-                    name: app.pendingInstall ? 'Installing...' : app.name,
+                    name: app.pendingInstall ? `Installing ${app.name}...` : app.name,
```

Comment:
Why change this? Installing + the app name will very rarely fit on the app tile and then the user will be left wondering what the heck it says haha. I chose Installing... because I figured they would remember which app they clicked install on or could tell by the icon and the app tile could still look clean.

Reply:
Just wanted the tiles to have different names that's all. Without this change, if you try to install multiple apps simultaneously(meaning multiple tile have the same value `Installing...`) it cause an error 
```js
Warning: flattenChildren(...): Encountered two children with the same key, `undefinedInstalling...`. Child keys must be unique; when two children share a key, only the first child will be used.
    in div (created by HScrollMenu)
    in HScrollMenu (created by AppStore)
    in div (created by AppStore)
    in AppStore (created by Connect(AppStore))
    in Connect(AppStore) (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in div (created by HMIApp)
    in div (created by HMIApp)
    in div (created by HMIApp)
    in HMIApp (created by Connect(HMIApp))
    in Connect(HMIApp)
...
```
It doesn't have to be the app name any unique identifier would be fine, the appName was just the most appropriate info to display to the user and the easiest to get.

Reply:
`Without this change,` fixing the error is just a side-effect of your change.

the key of the div is what matters, not the name shown in the UI: ```key={menuItem.cmdID + menuItem.name}```

Reply:
Oh I think I understand what you mean about the key. Fixed in https://github.com/smartdevicelink/generic_hmi/pull/241/commits/c16b4174d78769d08df67c10ae784653fbb23407